src/webgl/RendererOptions.ts

```diff
@@ -0,0 +1,49 @@
+import { Options, DefaultOptions, Viewport, Dimensions } from '../types/api'
+import { DEFAULT_OPTIONS, isNumber } from '../utils'
+
+export default class RendererOptions implements DefaultOptions {
```

Comment:
### RendererOptions
- straight forward class that manages updates renderer options
- automatically applies defaults
- single instance that can be a shared as a dependency without losing reference to the object after updates